### PR TITLE
OrderItem Class

### PR DESCRIPTION
**Pull request by okao21**
I fixed problems from pull request #17 and added a new enum class.

### CODE REVIEW

File: Ordering/Contracts/OrderManagement/OrderItem.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        public int ItemID { get; }
+        public string Name { get; set; }
+        public string ItemDescription { get; set; }
+        public double Price { get; set; }
+        public DateTime DateLastOrdered { get; set; }
+        public OrderItemCategory Category { get; set; }
+
+        public OrderItem(string name, string itemDescription, double price, DateTime date, OrderItemCategory cateogry)
+        {
+            ItemID = CurrentItemID;
```
Comment (baez) [inline Ordering/Contracts/OrderManagement/OrderItem.cs:23]: What will the ItemID represent, are you going to generate this ID inside this class?
If so, please explain based on what you will be generating it.
I suggest you discuss this with Jordan and consider where you will be needing this ID and write down the scenario to ensure this model works.

File: Ordering/Contracts/OrderManagement/OrderItem.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        public static int CurrentItemID = 0;
+
+        public int ItemID { get; }
+        public string Name { get; set; }
+        public string ItemDescription { get; set; }
+        public double Price { get; set; }
+        public DateTime DateLastOrdered { get; set; }
+        public OrderItemCategory Category { get; set; }
+
+        public OrderItem(string name, string itemDescription, double price, DateTime date, OrderItemCategory cateogry)
```
Comment (baez) [inline Ordering/Contracts/OrderManagement/OrderItem.cs:21]: is this order date? if so, rename to orderDate.

File: Ordering/Contracts/OrderManagement/OrderItem.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+{
+    public class OrderItem : IOrderItem
+    {
+        public static int CurrentItemID = 0;
+
+        public int ItemID { get; }
+        public string Name { get; set; }
+        public string ItemDescription { get; set; }
+        public double Price { get; set; }
+        public DateTime DateLastOrdered { get; set; }
```
Comment (baez) [inline Ordering/Contracts/OrderManagement/OrderItem.cs:18]: Why the date last ordered and not order date?

**Review by baez — CHANGES_REQUESTED** [review]

File: Ordering/Contracts/OrderManagement/OrderItem.cs (hunk 1 above)
Comment (okao21) [reply Ordering/Contracts/OrderManagement/OrderItem.cs:23]: I have declared a static variable called currentItemID. When an orderitem is initialized, we will assign orderitem's id with the currentItemID value and the currentItemID will increment by one. OrderItem's ID will always be unique when someone initializes orderItem variable.

File: Ordering/Contracts/OrderManagement/OrderItem.cs (hunk 3 above)
Comment (okao21) [reply Ordering/Contracts/OrderManagement/OrderItem.cs:18]: I have decided not to include this variable

**Pull request merged**